susy2015/AllHadronicSUSY
# New Search Bins (SBv3)

Hui updated SB2018.h to support SBv3 for both validation and search bins. 

Validation Bins
- use only < and >= for float values
- change high dm bin numbering to start from 19 instead of 22
- remove nJet >= 7 requirement for bins 23, 24, 25, 26

Search Bins
- use only < and >= for float values
- Low dm: no additional changes
- High dm: new version SBv3

Helper function for parsing selection strings.
- Used in ZInvisible, see PR: https://github.com/susy2015/ZInvisible/pull/214

Please see Hui's talk [here](https://indico.cern.ch/event/842046/#92-update-on-search-bin-study) for details.
